logger/logger.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import os
+
+from pubsub.pubsub import Publisher
+
+ENV = os.getenv("ENV")
+
+class PubSubLogHandler(logging.Handler):
+    def __init__(self)-> None:
+        self.publisher = Publisher()
+        logging.Handler.__init__(self=self)
+
+    def emit(self, record) -> None:
+        message = self.format(record)
+        self.publisher.publish(message)
+
+
+class PluginLogger:
+    def __init__(self) -> None:
+        self.logger = logging.getLogger("plugin_runner_logger")
+        self.logger.setLevel(logging.INFO)
+        formatter = logging.Formatter('%(levelname)s %(asctime)s %(message)s')
+
+        streaming_handler = logging.StreamHandler()
+        streaming_handler.setFormatter(formatter)
+        self.logger.addHandler(streaming_handler)
+
+        if ENV == "production":
```

Comment:
Instead of this, what do you think about using the presence of the REDIS_ENDPOINT env var? That's what really matters, right? "publish to redis if there is a redis to connect to"

Reply:
I like it - just added.